Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -605,7 +605,7 @@ function Add-GroupToCloudAdmins {
     elseif ($ExternalSources.count -eq 1) {
```

Comment:
What about the case where they have deliberately configured multiple identity sources?

Reply:
in that case the cmdlet attempts to search all the configured identity sources for the specified group name if a domain name is not provided.

if a domain name is provided, then it searches that specified domain for the group and returns an error if it's not in _that_ group (even if it is in a different identity source)